go.mod

```diff
@@ -5,10 +5,11 @@ go 1.13
 require (
 	github.com/gin-contrib/sse v0.1.0
 	github.com/go-playground/validator/v10 v10.4.1
+	github.com/goccy/go-json v0.5.1
```

Comment:
Why change this?

Reply:
It seems to be automatic generated when I run go mod tidy after upgrading github.com/ugorji/go/codec